src/sentry/incidents/endpoints/serializers/workflow_engine_action.py

```diff
@@ -34,17 +43,15 @@ def serialize(self, obj: Action, attrs, user, **kwargs):
             sentry_app_id = int(obj.config.get("target_identifier"))
             sentry_app_config = obj.data.get("settings")
 
+        trigger_action = AlertRuleTriggerAction.objects.get(id=aarta.alert_rule_trigger_action_id)
+
         result = {
-            "id": str(aarta.alert_rule_trigger_action.id),
-            "alertRuleTriggerId": str(aarta.alert_rule_trigger_action.alert_rule_trigger.id),
+            "id": str(aarta.alert_rule_trigger_action_id),
+            "alertRuleTriggerId": str(trigger_action.alert_rule_trigger.id),
```

Comment:
Is this also meant to be taken from one of the new models?

Reply:
yes agh I keep getting `AlertRuleTriggerAction` and `ActionAlertRuleTriggerAction` mixed up